creativecloud/features/firefly/firefly-masonry.css

```diff
@@ -0,0 +1,477 @@
+
+/* stylelint-disable property-no-vendor-prefix */
+/* stylelint-disable value-no-vendor-prefix */
+/* stylelint-disable selector-no-vendor-prefix */
+.ff-masonry .mobile-only,
+.ff-masonry .grid-layout {
+  display: none;
+}
+
+.masonry-prompttext::-moz-placeholder,
+.masonry-prompttext.light::-moz-placeholder {
+  font-style: italic;
+}
+
+.masonry-prompttext::placeholder,
+.masonry-prompttext.light::placeholder {
+  font-style: italic;
+}
```

Comment:
line 10 to 15 can be merged. Also make CSS relative

Reply:
done